utils/index.d.ts

```diff
@@ -880,6 +880,55 @@ export interface IWizardOptions<T extends IActionContext> {
     showLoadingPrompt?: boolean;
 }
 
+export interface ActivityTreeItemOptions {
+    label: string;
+    contextValuePostfix?: string;
```

Comment:
Still prefer the `contextValuesToAdd` shape...

Reply:
This just occurred to me, but the `contextValuesToAdd` approach could be difficult in scenarios where we want to change a specific state.  For example, if I `restart` an app, we add the context value `restarting`.  But when it resolves, it should have the context value of `running`.  In that case, how do we easily figure out that we need to remove the `restarting` context value as well?

I thought about this because I can imagine Alex was using this approach for similar reasons.

Reply:
That's a good point Nathan. We may need to expose the underlying `Set<string>`, with instruction to not remove things you didn't add.

Reply:
> This just occurred to me, but the contextValuesToAdd approach could be difficult in scenarios where we want to change a specific state. For example, if I restart an app, we add the context value restarting. But when it resolves, it should have the context value of running. In that case, how do we easily figure out that we need to remove the restarting context value as well?

I'm not sure I understand the problem, if the context value string is constructed in the context value getter, we'll always just add `contextValuesToAdd`. I'm going to leave it for now and we can discuss further.